spec/spec_helper.rb

```diff
@@ -1,16 +1,14 @@
-ENV['RACK_ENV'] = "test"
+ENV['RACK_ENV'] = 'test'
+
+Dir['./spec/support/**/*.rb'].sort.each { |f| require f}
 
-require 'rubygems'
-require 'bundler/setup'
 require 'mongoid'
```

Comment:
There is no need - Mongoid is specified as a dev/test dependency in `Gemfile`, so it will be available to anyone running the tests: https://github.com/simplybusiness/divisio/blob/support_ab_fab/Gemfile#L6-L9

> Don't know how big of a deal this is, because someone using the gem won't run the gem's test.

Exactly